checkov/main.py

```diff
@@ -131,6 +149,13 @@ def add_parser_args(parser):
     parser.add_argument('-b', '--branch',
                         help="Selected branch of the persisted repository. Only has effect when using the --bc-api-key flag",
                         default='master')
+    parser.add_argument('--skip-fixes',
```

Comment:
maybe we should add `bc` prefix. since this is only possible using API integration?

Reply:
We don't do that for repo and branch today, though.

Reply:
I don't think it's necessary, with the argument clearly documented (along with extra docs that we'll have in the readme and elsewhere).